tests/basics/struct1.py

```diff
@@ -121,3 +121,5 @@
 
 # check padding bytes
 print(struct.pack("xb", 3))
+# Make sure pack doesn't reuse a larger value and error
+print(struct.pack("xH", 0x100))
```

Comment:
Looks like tests are failing because there's no newline here
```suggestion
print(struct.pack("xH", 0x100))

```